Makefile

```diff
@@ -1,5 +1,8 @@
+all:
+	cd themes/qtail/; 
+	npm run build-qtail;
 public:
-	hugo --gc --minify
+	hugo --gc --minify;
 
 clean:
-	rm -rf public resources
+	rm -rf public resources themes/qtail/assets/style.css
```

Comment:
Can you also delete the checked-in copy in this commit?

And it looks like we'll probably need to change the github actions code to just do a `make`, so we don't have to encode this twice. That's in .github/workflows/... can you make that change in this commit as well?